http-server/README.md

```diff
@@ -71,3 +71,29 @@ wizard may fail to start with this error:
 java.lang.IllegalStateException: Unable to acquire the logger context
     at io.dropwizard.logging.LoggingUtil.getLoggerContext(LoggingUtil.java:46)
 ```
+
+## Stream is already closed
+
+This can happen when the server side does not fail fast on incorrect input.
+This can be if we use headers that are missing or do not check that parameters are present.
+
+The stack trace indicating this will look like this:
+```bash
+ERROR [2019-06-06 05:07:22,247] org.glassfish.jersey.server.ServerRuntime$Responder: An I/O error has occurred while writing a response message entity to the container output stream.
+! java.lang.IllegalStateException: The output stream has already been closed.
```

Comment:
Just curious: Would this issue occur when shutting down?
Like when the stream gets closed but there are still processes running that are ready to write, whenever there's a response?

Reply:
The exception happens while processing the request, at which point everything hangs, the request thread on the client as well as the server side processing thread. There is no response sent, and hence why it's a potential DDOS vector.

The hung thread does not terminate properly on shutdown, and so shutdown hangs and requires a force-kill.

It seems like the stream is closed early due to an exception but the server tries to write a response anyways and then just fails..